tests/unit/bokeh/core/property/test_visual.py

```diff
@@ -273,6 +274,20 @@ def test_transform_data_url(self) -> None:
         image = "data:image/png;base64,"
         assert prop.transform(image) == image
 
+    def test_transform_path(self) -> None:
+        temp_dir = Path(tempfile.gettempdir())
+        filename = tempfile.NamedTemporaryFile().name
+        location = temp_dir / filename
+        image = PIL.Image.new('RGBA', size=(50, 50), color=(155, 0, 0))
+        image.save(location, 'png')
+        prop = bcpv.Image()
+        assert prop.transform(location).startswith("data:image/png")
```

Comment:
```suggestion
        temp_file = tempfile.NamedTemporaryFile()
        location = temp_dir / temp_file.name
        image = PIL.Image.new('RGBA', size=(50, 50), color=(155, 0, 0))
        image.save(location, 'png')
        prop = bcpv.Image()
        assert prop.transform(location).startswith("data:image/png")
        temp_file.close()
```

This test fails at the moment because of an ResourceWarning. You have to close the `NamedTemporaryFile` after usage.

See https://docs.python.org/3/library/tempfile.html#examples

Reply:
@mosc9575 Thanks for pointing øut the error